spec/lib/html_cleaner_spec.rb

```diff
@@ -1147,56 +1146,22 @@
     end
 
     context "with keep_src: false" do
```

Comment:
The new regex definitely doesn't care about the quote formatting anymore, but maybe it'd be good to keep these tests to show that?